presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -897,14 +897,16 @@ public long getRunAwaySplitCount()
         private final long startTime;
         private final String threadId;
         private final Thread thread;
+        private final PrioritizedSplitRunner split;
         private boolean printed;
 
-        public RunningSplitInfo(long startTime, String threadId, Thread thread)
+        public RunningSplitInfo(long startTime, String threadId, Thread thread, PrioritizedSplitRunner split)
         {
             this.startTime = startTime;
             this.threadId = threadId;
             this.thread = thread;
             this.printed = false;
+            this.split = split;
```

Comment:
move this above `this.printed = false;` to align with stack order

Reply:
done